docs/setup/compose_based.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Compose Based Deployment
+---
+
+<head>
+  <title>Environment Setup</title>
+  <meta
+    name="description"
+    content="Here we'll deploy our pipeline in the Docker Container"
+  />
+</head>
+
+Here we'll deploy our pipeline in the Docker Container using compose.
+
+## Pre-requisites:
+- Jenkins Server for running pipelines
+- Hashicorp Vault for storing secrets
+- Docker Swarm (This could be your localhost as well)
+- Make cli for running Makefile
+
+## Setting up the Pipeline
+- ### **Clone the repository**
```

Comment:
Please remove every instruction associated with vault, jenkins etc. We don't need to install them.